engine/time/src/main/java/io/deephaven/time/DateTime.java

```diff
@@ -154,9 +202,7 @@ public org.joda.time.DateTime getJodaDateTime(TimeZone timeZone) {
      * @return a Java Instant representing this DateTime
      */
     public Instant getInstant() {
-        long epochSecond = nanos / 1000000000;
-        long nanoAdjustment = nanos % 1000000000;
-        return Instant.ofEpochSecond(epochSecond, nanoAdjustment);
+        return Instant.ofEpochSecond(0, nanos);
```

Comment:
This looked correct before.

Reply:
This is a slight improvement, we don't have to do the math. ourselves. The internals of ofEpochSecond can deal directly w/ `long nanos`.